packages/studio/package.json

```diff
@@ -10,54 +10,23 @@
     "dev": "tsc --watch --preserveWatchOutput -p tsconfig.node.json & tsc --watch --preserveWatchOutput -p tsconfig.json",
     "build": "run-script-os",
     "build:default": "npm run build:windows && chmod 755 lib/bin/studio.js",
-    "build:windows": "rimraf lib && tsc -p tsconfig.node.json && tsc -p tsconfig.json",
-    "preview": "vite preview",
-    "test": "jest",
-    "typecheck-jest": "npx tsc -p tests/tsconfig.json"
+    "build:windows": "rimraf lib && tsc -p tsconfig.node.json && tsc -p tsconfig.json"
   },
   "dependencies": {
-    "@dhmk/zustand-lens": "^2.0.5",
-    "@minoru/react-dnd-treeview": "^3.4.1",
-    "@restart/ui": "^1.5.2",
     "@vitejs/plugin-react": "^4.0.4",
     "@yext/studio-plugin": "0.22.0",
+    "@yext/studio-ui": "0.22.0",
```

Comment:
to check my understanding, we'll now be publishing `studio-plugin`, `studio-ui`, and `studio` to npm, but user's should only have to install `studio` to run Studio with their repo?

Reply:
👍

Reply:
can we update the `update-minor-versions` script to bump `studio-ui` too?